webapp/channels/src/components/post_edit_history/restore_post_history.scss

```diff
@@ -0,0 +1,8 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+#restorePostModal {
+    .modal-content {
+        width: 832px;
```

Comment:
How does this work on different screen sizes? I don't know if we even support the post edit history on mobile view, but that doesn't kick in until below these dimensions, so this seems like it'll still overflow before we get to mobile view

Reply:
Yeah it wasn't looking right in narrow windows. Fixed it and now it looks good-

![Screenshot 2025-01-09 at 10 02 06 AM](https://github.com/user-attachments/assets/7eca96b0-6ba1-4045-a6d5-719658a393a0)
![Screenshot 2025-01-09 at 10 02 13 AM](https://github.com/user-attachments/assets/f0768614-742f-4f9b-80fc-ce6dd1fbd44d)
![Screenshot 2025-01-09 at 10 02 22 AM](https://github.com/user-attachments/assets/37c10876-08bc-402f-a77c-a1f6eb96dff0)
![Screenshot 2025-01-09 at 10 02 31 AM](https://github.com/user-attachments/assets/48b6b093-67e1-4e42-9458-e203e2d5af73)